tf-modules/archive/async_operation.tf

```diff
@@ -45,3 +50,46 @@ resource "aws_ecs_task_definition" "async_operation" {
 ]
 EOS
 }
+
+resource "aws_ecs_task_definition" "dead_letter_recovery_operation" {
+  depends_on               = [aws_cloudwatch_log_group.async_operation]
+  family                   = "${var.prefix}-AsyncOperationTaskDefinition"
+  tags                     = var.tags
+  requires_compatibilities = ["FARGATE"]
+  network_mode             = "awsvpc"
+  execution_role_arn       = var.ecs_execution_role.arn
+  task_role_arn            = var.ecs_task_role.arn
+  cpu                      = var.dead_letter_recovery_cpu
+  memory                   = var.dead_letter_recovery_memory
+  container_definitions    = <<EOS
+[
+  {
+    "name": "AsyncOperation",
+    "essential": true,
+    "environment": [
+      {
+        "name": "AWS_REGION",
+        "value": "${data.aws_region.current.name}"
+      },
+      {
+        "name": "databaseCredentialSecretArn",
+        "value": "${var.rds_user_access_secret_arn}"
+      },
+      {
+        "name": "ES_HOST",
+        "value": "${var.elasticsearch_hostname}"
+      }
+    ],
+    "image": "${var.async_operation_image}",
+    "logConfiguration": {
+      "logDriver": "awslogs",
+      "options": {
+        "awslogs-group": "${aws_cloudwatch_log_group.dead_letter_recovery.name}",
+        "awslogs-region": "${data.aws_region.current.name}",
+        "awslogs-stream-prefix": "async-operation"
```

Comment:
dead_letter_recovery if we want seperate log group prefix.

Reply:
Thanks for catching the inconsistency in the group names/etc.    1f7e8957b33